### PR TITLE
chore(app-rust): improve rust app dev exp & ignore build folder

### PR DESCRIPTION
**Pull request by cbrzn**
this PR aims to improve the development experience with rust codegen:
- removes the `Module` word from imported modules
- update how the invoke overwrites are passed, now, an `InvokeOptions` struct has been created
- imported `env` was missing, now its being correctly fetched

### CODE REVIEW

File: implementations/app-rust/src/templates/types_rs.rs [hunk 1]
```diff
@@ -93,48 +100,67 @@ pub enum {{detect_keyword (to_upper type)}} {
...
+    pub uri: Uri,
+    pub invoker: Arc<dyn Invoker>,
+    pub env: Option<Vec<u8>>
 }
 
-impl {{detect_keyword (to_upper type)}} {
-    pub fn new(uri: Option<Uri>, invoker: Option<Arc<dyn Invoker>>, env: Option<Vec<u8>>) -> {{detect_keyword (to_upper type)}} {
+impl {{remove_module_suffix (detect_keyword (to_upper type))}} {
+    pub fn new(invoke_options: Option<InvokeOptions>) -> {{remove_module_suffix (detect_keyword (to_upper type))}} {
         let mut config = PolywrapClientConfig::new();
```
Comment (krisbitney) [inline implementations/app-rust/src/templates/types_rs.rs:119]: I know this was here before and isn't a new addition from you, but I think we should be lazily instantiating the default PolywrapClientConfig and PolywrapClient. What do you think?

**Review by krisbitney — CHANGES_REQUESTED** [review]
Really fantastic work. Thanks for doing this!!

File: implementations/app-rust/src/templates/types_rs.rs (hunk 1 above)
Comment (cbrzn) [reply implementations/app-rust/src/templates/types_rs.rs:119]: yah that make sense; will do it

**Review by krisbitney — APPROVED** [review]
LGTM!

**Review by dOrgJelli — APPROVED** [review]

**Pull request merged**